setup.py

```diff
@@ -95,7 +94,6 @@
     'invenio-workflows~=7.0,>=7.0.1',
     'langdetect~=1.0,>=1.0.7',
     'latexcodec~=1.0,>=1.0.5',
-    'librabbitmq~=1.0,>=1.6.1',
```

Comment:
This should still be supported (quoting http://docs.celeryproject.org/en/latest/userguide/configuration.html#broker-url):

> The transport part is the broker implementation to use, and the default is amqp, (uses librabbitmq if installed or falls back to pyamqp). There are also other choices available, including; redis://, sqs://, and qpid://.

Reply:
Mh, uninstalling `librabbitmq` is recommended by Celery's creator: https://github.com/celery/celery/issues/3675#issuecomment-266596324.

Reply:
I am reading that differently: the person is explicitly asking for `pyamqp`, with `librabbitmq` installed. From the docs, I am gathering that you can use the `amqp` protocol to use either, but if you are explicitly asking for `pyamqp`, you shouldn't have `librabbitmq` installed. Otherwise, it would completely contradict the official docs in several places.

Reply:
You were right here and above about `pyamqp` ! Amended the PR.

Reply:
Indeed, despite what the docs say, `pyamqp` is recommended by the celery people: https://github.com/celery/librabbitmq/issues/118#issuecomment-385461136